templates/experiments/experiment_session_view.html

```diff
@@ -40,6 +40,20 @@ <h3 class="text-base font-semibold leading-7 mt-8 pl-4">Experiment details</h3>
     <div class="max-w-5xl mt-4 border rounded-2xl">
       {% include "experiments/components/experiment_details.html" %}
     </div>
+    {% if experiment_session.get_participant_data %}
+      <div x-data="{showParticipantData: false}">
```

Comment:
just wondering if an Accordion widget would work instead of using alpine: https://daisyui.com/components/accordion/